adb/contrib/high.py

```diff
@@ -146,8 +146,8 @@ def _InitCache(device):
         '/sys/devices/system/cpu/cpu0/cpufreq/scaling_available_governors')
     if out:
       available_governors = sorted(i for i in out.split())
-      assert set(available_governors).issubset(
-          KNOWN_CPU_SCALING_GOVERNOR_VALUES), available_governors
+      assert len(set(available_governors).intersect(
```

Comment:
Actually, len() and > 0 are not needed.

This will work:
assert set(available_governors).intersect(KNOWN_CPU_SCALING_GOVERNOR_VALUES), available_governors